wdl-gauntlet/src/lib.rs

```diff
@@ -228,27 +228,27 @@ pub async fn gauntlet(args: Args) -> Result<()> {
                         .context("failed to write diagnostic")?;
 
                     assert!(
-                        actual.insert(
+                        actual.insert((
+                            diagnostic.labels().next().map(|l| l.span().start()),
```

Comment:
We'll still need to use `line_index` from the above `SimpleFile` to get the line number as `start` is returning the byte offset into the file.